config/session.php

```diff
@@ -36,6 +36,21 @@
 
     'expire_on_close' => env('SESSION_EXPIRE_ON_CLOSE', false),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Maintenance Bypass Cookie Lifetime
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify the number of minutes the maintenance mode
+    | bypass cookie should remain valid. This cookie is issued when a
+    | user accesses the application using the secret bypass URL. Once
+    | set, it allows temporary access to the application while it is
+    | in maintenance mode. The default duration is 12 hours.
```

Comment:
Small suggestion to keep the style of comments the same troughout the file (3 lines becoming shorter): 

```suggestion
    | When maintenance mode is enabled, a bypass cookie will be issued when a
    | user accesses the application using the bypass URL. This cookie will
    | allow access to the user for the configured duration in minutes.    
```